docker/Dockerfile

```diff
@@ -27,17 +27,19 @@ RUN cd /usr/lib/python3.9 && tar -zxvf /python-libs.tgz
 RUN echo "host replication replication all scram-sha-256" >> /usr/share/postgresql/14/pg_hba.conf.sample
 
 FROM server as backup
+COPY --from=lovelysystems/adminbox:0.4.2 /usr/bin/notify_if_fails /usr/bin/slack /usr/bin/
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-    cron barman barman-cli barman-cli-cloud && \
+    cron jq curl barman barman-cli barman-cli-cloud && \
     apt-get clean
 
 COPY psqlrc /root/.psqlrc
 
 ENV PGHOST=${PGHOST:-127.0.0.1} \
     PGUSER=${PGUSER:-postgres} \
-    BARMAN_BACKUP_SCHEDULE="0 4 * * *"
+    BARMAN_BACKUP_SCHEDULE="0 4 * * *" \
+    NOTIFY_BACKUP_ERROR_MESSAGE="Backup failed"
```

Comment:
using $PGHOST or $PGUSER here results in empty string. Not sure how to evauate these environment vars during build.

Reply:
at least locally in bash this syntax should work
```
frisi@frisi-xps:~$ export PGHOST=localhost
frisi@frisi-xps:~$ echo "hallo ${MESSAGE:-backup failed for $PGHOST}"
hallo backup failed for localhost
frisi@frisi-xps:~$ export MESSAGE="custom message"
frisi@frisi-xps:~$ echo "hallo ${MESSAGE:-backup failed for $PGHOST}"
hallo custom message
```